manubot/cite/citekey.py

```diff
@@ -1,38 +1,7 @@
 import functools
 import logging
-import re
```

Comment:
This file would be a lot easier to review if the functions were not reordered. Any reason we can't keep the existing order for now?

Reply:
The import recording here is good, since the diff is simple and we try to keep import alphabetical.

But is the reordering below of whole functions necessary? It makes it harder to review, so unless the new order is highly preferable I think it makes sense to keep the function definition order from manubot/cite/util.py since the diff detects this as the moved file. Would help tremendously with review. I've run `git diff --color-moved=dimmed-zebra master` locally but there it's still a challenge to mentally parse.